views/includes/inputs/checkbox-input.html

```diff
@@ -0,0 +1,16 @@
+{{ govukCheckboxes({
```

Comment:
These look like common widget HTML definitions. Could/should they be used by all the existing forms too or are they specific to trusts?